openmdao/docs/features/warnings.rst

```diff
@@ -0,0 +1,93 @@
+.. _feature_warnings:
+
+***************
+Warning Control
+***************
+
+OpenMDAO has several classes of warnings that may be raised during operation.
+In general, these warnings are useful and the user should pay attention to them.
+Sometimes these warnings can be unnecessarily noisy.
+Filtering out noisy "low-priority" warning can make other more important ones more obvious.
+
+OpenMDAO-Specific Warning Categories
+------------------------------------
+
+Class **OpenMDAOWarning** serves as the base-class for all OpenMDAO-specific warnings.
+All OpenMDAO-specific warnings default to a filter of 'always'.
+The following table shows all OpenMDAOWarning-derived classes.
+
++-----------------------------+--------------------------------------------------------------------------------------------+
+| Warning Class               | Description                                                                                |
++=============================+============================================+===============================================+
+| CacheWarning                | A cache is invalid and must be discarded.                                                  |
```

Comment:
Descriptions of these warnings should be made consistent, i.e., some start with "Warning (issued/encountered) (when/by) ..." and others don't.

Reply:
resolved